test/Modules/ModulesGraded.jl

```diff
@@ -1030,8 +973,7 @@ function randpoly(R::Oscar.Ring,coeffs=0:9,max_exp=4,max_terms=8)
 end
 
 @testset "Basic degree and homogeneity tests for free modules" begin
-    Qx, x = polynomial_ring(QQ, 3)
-    R, x = grade(Qx)
+    R, x = graded_polynomial_ring(QQ, 3)
```

Comment:
Maybe revert this change. The corresponding signature is not available and you might just resolve the problem with you future PR which will contain a unified solution.